src/ui/src/app/chart-search/chart-search.component.ts

```diff
@@ -1,56 +0,0 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
Why did you remove these files? The purpose of `search` components is to create a specific view with an independent URL. Users can link to an specific search.